### ISSUE TITLE
The 1901 TNA Civil Parish extraction is meaningless

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#1124]
Following video discussions with @geoffj-FUG  we have formed the opinion that the TNA civil parish extraction is effectively meaningless at the civil parish level.
For example Blagdon piece in Somerset is extracted as
![blagdon.png](https://images.zenhubusercontent.com/5b68ae684b5806bc2bdefcf0/058ac073-2127-4f39-b6f1-cf47dca8c265)
This is from a TNA record of 
![tna blagdon.png](https://images.zenhubusercontent.com/5b68ae684b5806bc2bdefcf0/7825d1b0-be33-48c3-95df-20f2502c169d)

The issue is that list includes civil parish, township or place, as the chapeau states 

Because they have included all of the "place" in the explanation of the area covered in the enumeration district.
![cover.png](https://images.zenhubusercontent.com/5b68ae684b5806bc2bdefcf0/95a28f2d-921c-4838-a282-141d6fee5779)

Since we have no way of distinguishing what is what from that list we have extracted allas civil parishes. This is clearly incorrect BUT we have no alternative.
In conclusion the civil parish extraction is effectively meaningless.

**Comment by Captainkirkdawson** [comment]
We propose the following:
System actions 1)update the civil parishes list to 1901 to eliminate all entries.  2) provide access to the existing action of adding a civil parish to county and master county coordinators
Documentation. The instructions for the transcriptions of 1901 be updated to reflect that the coordinator when starting work on a new 1901 piece they review the Enumeration District  documents to identify the civil parishes and adds them to the piece.
All pieces will be impacted.
